.github/workflows/go.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         GO_VERSION:
           - "1.16.x"
           - "1.20.x"
-          - "1.21.0-rc.2"
+          - "1.21.x"
```

Comment:
Go ahead and remove go 1.20.x both here and below. We were only testing both 1.20 and 1.21rc to check that updating to go1.21 wouldn't break anything. Now that it's time to do the update for real, we can leave go 1.20 behind.

Reply:
@aarongable 
fixed ,and test passed
https://github.com/go-jose/go-jose/pull/54/commits/94a19933299e56103245c6d76cc462d94a4a0475